openmetadata-ui/src/main/resources/ui/src/components/AddIngestion/Steps/ConfigureIngestion.tsx

```diff
@@ -234,15 +234,15 @@ const ConfigureIngestion = ({
               {getSeparator('')}
             </Field>
           )}
-          {!isNil(markDeletedTablesFromFilterOnly) && (
+          {!isNil(markAllDeletedTables) && (
             <Field>
               <div className="tw-flex tw-gap-1">
                 <label>Mark Deleted Tables from Filter Only</label>
```

Comment:
It should be 
`Mark All Deleted Tables`

Reply:
Done